src/components/Popover.re

```diff
@@ -0,0 +1,61 @@
+[@bs.new] [@bs.module "react-tiny-popover"]
+external make:
+  {
+    .
+    "containerClassName": string,
+    "isOpen": bool,
+    "align": string,
+    "padding": int,
+    "content": React.element,
+    "position": list(string),
+    "transitionDuration": float,
+    "onClickOutside": unit => unit,
+    "children": React.element,
+  } =>
+  React.element =
+  "default";
+
+[@bs.obj]
+external makeProps:
```

Comment:
not needed to write the makeProps yourself! you can just put it above and add the `[@react.component]` after `bs.module`

Reply:
https://github.com/Astrocoders/last-card/blob/develop/packages/core/src/types/RNTags.re#L2